flink-state-backends/flink-statebackend-forst/src/test/java/org/apache/flink/state/forst/ForStStateExecutorTest.java

```diff
@@ -120,6 +120,113 @@ public void testExecuteValueStateRequest() throws Exception {
+            assertThat(((TestStateFuture<Integer>) request.getFuture()).getCompletedResult())
+                    .isEqualTo(Integer.parseInt(key));
+        }
+
+        // 3. add more value for the aggregate state
+        stateRequestContainer = forStStateExecutor.createStateRequestContainer();
+        int addCnt = 10;
+        for (int i = 0; i < keyNum; i++) {
+            for (int j = 0; j < addCnt; j++) {
+                StateRequest<String, Integer, Integer> r =
+                        (StateRequest<String, Integer, Integer>)
+                                buildStateRequest(
+                                        state, StateRequestType.AGGREGATING_PUT, "" + i, 1, i * 2);
+                requests.add(r);
+                stateRequestContainer.offer(r);
+            }
+        }
+
+        // clear first 100 state
+        for (int i = 0; i < 100; i++) {
+            StateRequest<String, Integer, Integer> r =
+                    (StateRequest<String, Integer, Integer>)
+                            buildStateRequest(
+                                    state, StateRequestType.CLEAR, "" + i, null, "record" + i);
+            requests.add(r);
+            stateRequestContainer.offer(r);
+        }
+
+        forStStateExecutor.executeBatchRequests(stateRequestContainer).get();
+
+        // 4. read and verify the updated aggregate state
+        requests = new ArrayList<>();
+        stateRequestContainer = forStStateExecutor.createStateRequestContainer();
+        for (int i = 0; i < keyNum; i++) {
+            StateRequest<String, Integer, Integer> r =
+                    (StateRequest<String, Integer, Integer>)
+                            buildStateRequest(
+                                    state,
+                                    StateRequestType.AGGREGATING_GET,
+                                    "" + i,
+                                    null,
+                                    "record" + i);
+            requests.add(r);
+            stateRequestContainer.offer(r);
+        }
+
+        forStStateExecutor.executeBatchRequests(stateRequestContainer).get();
+
+        for (int i = 0; i < 100; i++) {
+            StateRequest<String, Integer, Integer> request = requests.get(i);
+            assertThat(request.getRequestType()).isEqualTo(StateRequestType.AGGREGATING_GET);
+            assertThat(((TestStateFuture<Integer>) request.getFuture()).getCompletedResult())
+                    .isEqualTo(null);
+        }
+        for (int i = 100; i < keyNum; i++) {
+            StateRequest<String, Integer, Integer> request = requests.get(i);
+            assertThat(request.getRequestType()).isEqualTo(StateRequestType.AGGREGATING_GET);
+            String key = request.getRecordContext().getKey();
+            assertThat(request.getRecordContext().getRecord()).isEqualTo("record" + key);
+            assertThat(((TestStateFuture<Integer>) request.getFuture()).getCompletedResult())
+                    .isEqualTo(1);
```

Comment:
IIUC, the result should be equal to `key+1`, could you please illustrate why this is equal to `1`?